pygpcca/_gpcca.py

```diff
@@ -971,12 +975,20 @@ def optimize(
                     f"Can't check complex conjugate block splitting for {m} clusters with only "
                     f"{len(self._p_eigenvalues)} eigenvalues."
                 )
-            else:
-                if _check_conj_split(self._p_eigenvalues[:m]):
+            if _check_conj_split(self._p_eigenvalues[:m]):
+                if len(m_list) == 1:
                     raise ValueError(
-                        f"Clustering into {m} clusters will split conjugate eigenvalues. "
+                        f"Clustering into {m} clusters will split complex conjugate eigenvalues. "
                         f"Request one cluster more or less."
                     )
+                warnings.warn(
+                    f"Clustering into {m} clusters will split complex conjugate eigenvalues. "
+                    f"Skipping clustering into {m} clusters."
+                )
+                crispness_list.append(0)
+                chi_list.append(np.zeros((n, m)))
+                rot_matrix_list.append(np.zeros((m, m)))
```

Comment:
Would NaN's be better here, to indicate that this hasn't worked?

Reply:
For the `crispness` not, since it is defined as `xi in [0,1]` with 0 indicating no clustering (which matches here) and 1 indicating perfectly sharp clustering.
With `chi` and `rot_matrix` I'm undecided.
@michalk8 what is your feeling here?

Reply:
I'd go with NaN's as well, since interval `[0, 1]` is valid.

Reply:
Done [here](https://github.com/msmdev/pyGPCCA/pull/13/commits/a2c47c9a7a2fd880540e758067a68b53d762b8f6)!